tsconfig.node.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["*.config.ts", "*.config.js", "*.config.mjs", "eslint.config.mjs"]
+}
```

Comment:
node tsconfig ? for a static website ? is this for playwright ?

Reply:
it's for the linter